build-support/bin/ci.sh

```diff
@@ -235,6 +237,15 @@ if [[ "${run_rust_tests:-false}" == "true" ]]; then
   end_travis_section
 fi
 
+if [[ "${run_cargo_audit:-false}" == "true" ]]; then
+  start_travis_section "CargoAudit" "Running cargo audit on rust code"
+  (
+    "${REPO_ROOT}/build-support/bin/native/cargo" ensure-installed --package=cargo-audit --version=0.5.2
+    "${REPO_ROOT}/build-support/bin/native/cargo" audit -f "${REPO_ROOT}/src/rust/engine/Cargo.lock"
+  ) || die "Pants clippy failure"
```

Comment:
nit: `s/Pants clippy failure/Pants cargo-audit failure/`

Reply:
^ This is still a thing.